.github/workflows/compile.yml

```diff
@@ -16,3 +16,14 @@ jobs:
         run: |
           docker compose -f ./docker-compose-1.yml build --no-cache
           docker compose -f ./docker-compose-2.yml build --no-cache
+
+          docker compose -f ./docker-compose-3.yml build wildfly-logstash-release --no-cache
+          docker compose -f ./docker-compose-3.yml build wildfly-jpa-logstash-release --no-cache 
+          docker compose -f ./docker-compose-3.yml build wildfly-full-logstash-release --no-cache
+
+          docker compose -f ./docker-compose-3.yml build wildfly-oracle-jdbc-release --no-cache
+          docker compose -f ./docker-compose-3.yml build wildfly-full-oracle-jdbc-release --no-cache
+          docker compose -f ./docker-compose-3.yml build wildfly-postgresql-jdbc-release --no-cache
+          docker compose -f ./docker-compose-3.yml build wildfly-full-postgresql-jdbc-release --no-cache
+
+          docker compose -f ./docker-compose-3.yml build wildfly-full-oracle-jdbc-ibm-mq-release --no-cache
```

Comment:
ez ha jól sejtem a parallel build miatt kell, depends on-t meg lehetne próbálni a yml-ben és akkor a workflowba elég docker compose -f ./docker-compose-3.yml build --no-cache

Reply:
paralell miatt, lehet amugy itt mukodne is siman a build, mert a compose meg tudja kezelniu a depends_on-t de a bake mar nem. kiprobalom